components/brave_extension/extension/brave_extension/background/events/cosmeticFilterEvents.ts

```diff
@@ -45,6 +46,32 @@ chrome.runtime.onMessage.addListener((msg, sender, sendResponse) => {
       rule.host = msg.baseURI
       break
     }
+    case 'classIdStylesheet': {
+      chrome.braveShields.classIdStylesheet(msg.classes, msg.ids, msg.exceptions, stylesheet => {
+        chrome.tabs.insertCSS({
+          code: stylesheet,
+          cssOrigin: 'user',
+          runAt: 'document_start'
+        })
+      })
```

Comment:
`)` should be on separate line since `{` is separate line to `(`

Reply:
Hmm, not sure what you mean? The first `})` matches to `...insertCSS({`, the second matches to `...Stylesheet(... => {`.